lib/cli/behaviour/defaults.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+  dir: process.cwd(),
+  force: false,
+  name: 'default-behaviour',
```

Comment:
I don't think there's value in this default. I'd make the default be the name of the folder in which the command is being run.

The most common workflow will be:

* create new git repo
* clone empty git repo
* `cd <repo name>`
* `hof behaviour .`

So if the default name is the directory name, then it will match the repo name, which I think is the best approach and mostly likely desired behaviour.